rfqmsg/messages.go

```diff
@@ -76,6 +79,10 @@ var (
 	// ErrUnknownMessageType is an error that is returned when an unknown
 	// message type is encountered.
 	ErrUnknownMessageType = errors.New("unknown message type")
+
+	// MilliSatPerBtc is the number of milli-satoshis in a bitcoin.
+	// This is 100 billion, which is 10^11.
+	MilliSatPerBtc = NewUint64FixedPoint(1, 11)
```

Comment:
Perhaps consider the name `MsatPerBtc` instead? I think it's shorter but still clear.